app-text/ghostscript_gpl/ghostscript_gpl-9.24.recipe

```diff
@@ -12,48 +12,47 @@ SOURCE_URI="https://github.com/ArtifexSoftware/ghostpdl-downloads/releases/downl
 CHECKSUM_SHA256="e613b2376f01882e82e0cf08f6caa3c5719793c4e31dd7d4a8e87d61d371005f"
 SOURCE_DIR="ghostscript-$portVersion"
 
-ARCHITECTURES="?x86_gcc2 ?x86 x86_64"
-SECONDARY_ARCHITECTURES="x86"
+ARCHITECTURES="!x86_gcc2 !x86 x86_64"
+SECONDARY_ARCHITECTURES="!x86"
```

Comment:
Oops, for `9.24` we need to mark x86 as tested. While at it, could you also add the test which only adds x86 secondary arch for x86_gcc2h? Here's what I have in mind:
```bash
ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
if [ "$targetArchitecture" = x86_gcc2 ]; then
SECONDARY_ARCHITECTURES="x86"
fi
```

BTW, I guess we don't need to bump `REVISION` because:
1) the changes are neutral for x86_64 and x86 primary arch,
2) and the package was probably not building on x86 **secondary** arch.

But if a binary package for ghostscript_gpl_x86 9.24-1 already exists then we should bump `REVISION` to `2`.